emscripten.py

```diff
@@ -301,7 +301,7 @@ def emscript(in_wasm, out_wasm, outfile_js, memfile, DEBUG):
   # memory and global initializers
 
   if settings.RELOCATABLE:
-    static_bump = align_memory(webassembly.parse_dylink_section(in_wasm)[0])
+    static_bump = align_memory(webassembly.parse_dylink_section(in_wasm)[0].mem_size)
```

Comment:
does this still need the `[0]` or is that supposed to be replaced by `.mem_size`?

Reply:
WTF.. how did that get past CI..

Reply:
Remove the second return value.. it was needed for a prior caller that has since been removed.